modules/lb_internal/variables.tf

```diff
@@ -42,8 +42,19 @@ variable "subnetwork" {
   type = string
 }
 
+variable "ip_version" {
+  description = "(Optional) The IP version that will be used by this Load Balancer. Possible values are: IPV4 (default), IPV6."
+  type        = string
+  default     = null
+}
+
 variable "ip_address" {
```

Comment:
Can we please also add the `type` of the variable here

Reply:
No problem!